pkg/api/api.go

```diff
@@ -437,7 +437,8 @@ func (hs *HTTPServer) registerRoutes() {
 	r.Delete("/api/snapshots/:key", reqEditorRole, Wrap(DeleteDashboardSnapshot))
 
 	// Health check
-	r.Get("/api/health", hs.healthHandler)
+	r.Get("/api/health", hs.apiHealthHandler)
+	r.Get("/healthz", hs.healthzHandler)
```

Comment:
is this a common naming for this type of url endpoint? The other option I had was to just have this be a param for the /api/health endpoint `?ignore_db`  , feels messy to have two health endpoints

Reply:
I was thinking also about having an extra parameter in the existing endpoint but I don't know since it's served under a different path (not under /api).

Reply:
`healthz` or `healthy` is kind the standard name for health checks in the Kubernetes world. I prefer `healthz` since its the likelihood of a collision is very low. But we could put them on another route. Prometheus has all these kinds of routes under `/-/` so `/-/healthz` for example. 

The `/api/health` endpoint could introduce more checks, ex checking that it can reach all configured datasources so I think it would be nice to have separate endpoints.

Reply:
`/healthz` is definitely a convention that many people follow. Ex

https://github.com/healthz
https://grafana.com/healthz

https://vimeo.com/173610242

@torkelo unless you object I would like to merge this for 7.3 so we can avoid using robots.txt in HG :)

Reply:
no objection! merge this baby!